tests/app/celery/test_process_ses_receipts_tasks.py

```diff
@@ -311,6 +311,7 @@ def test_ses_callback_should_set_status_to_temporary_failure(client,
     assert get_notification_by_id(notification.id).status == 'sending'
     assert process_ses_receipts_tasks.process_ses_results(ses_soft_bounce_callback(reference='ref'))
     assert get_notification_by_id(notification.id).status == 'temporary-failure'
+    assert get_notification_by_id(notification.id).status_reason == "Temporarily failed to deliver email due to soft bounce"  # noqa: E501
```

Comment:
what does `# noqa: E501` mean?

Reply:
This removes the validation of the line length, since it's rather messy otherwise. I already went ahead and merged since you had previously approved the PR.